docs/core/diagnostics/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: .NET Core Diagnostics
+description: An overview of the tools and APIs available to diagnose .NET Core applications.
+author: stmaclea
```

Comment:
author is the GitHub alias and ms.author the MS alias, so here and in the other articles should be 
```suggestion
author: sdmaclea
```